Because xmlto-0.0.22 adds --noent to xmllint options, it's more strict,
and doesn't validate anymore. This is a temp. solution until upstream
gets their documentation fixed.

http://bugs.gentoo.org/show_bug.cgi?id=271678

diff -ur ConsoleKit-0.3.0.orig/doc/dbus/Makefile.am ConsoleKit-0.3.0/doc/dbus/Makefile.am
--- ConsoleKit-0.3.0.orig/doc/dbus/Makefile.am	2008-07-31 00:35:51.000000000 +0300
+++ ConsoleKit-0.3.0/doc/dbus/Makefile.am	2009-06-09 22:30:57.000000000 +0300
@@ -16,7 +16,7 @@
 htmldoc_DATA = ConsoleKit.html
 
 ConsoleKit.html : $(SPEC_XML_FILES)
-	$(XMLTO) xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
+	$(XMLTO) --skip-validation xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
 
 endif # DOCBOOK_DOCS_ENABLED
 
